packages/docusaurus-theme-classic/src/theme/Footer/Logo/index.tsx

```diff
@@ -31,8 +32,17 @@ function LogoImage({logo}: Props) {
 }
 
 export default function FooterLogo({logo}: Props): JSX.Element {
+  let logoLinkProps = {};
+  if (logo.target) {
+    logoLinkProps = {target: logo.target};
+  } else if (!isInternalUrl(logo.href)) {
+    logoLinkProps = {
+      rel: 'noopener noreferrer',
+      target: '_blank',
+    };
+  }
```

Comment:
This branch not necessary. Outbound links already get these props. We have this logic in the `Link` component.